scripts/unify

```diff
@@ -13,37 +13,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const async = require(`async`);
 const fs = require(`fs`);
 const path = require(`path`);
 
-const PROJECT_ROOT = path.dirname(__dirname);
+const PROJECT_ROOT = path.join(__dirname, `..`);
 
 // Get subdirectories with a `package.json` file
 const directories = fs.readdirSync(PROJECT_ROOT)
-                      .filter(dir => fs.existsSync(`${PROJECT_ROOT}/${dir}/package.json`));
+                      .filter((dir) => fs.existsSync(path.join(PROJECT_ROOT, dir, `package.json`)));
 
 // Dedupe package.json dependencies
 // WARNING: This will fail if two different versions of the same package are required.
 let pkgSet = {};
-let devPkgSet = {};
 let pkgJson;
-for (const dir of directories) {
-  pkgJson = JSON.parse(fs.readFileSync(`${PROJECT_ROOT}/${dir}/package.json`));
-
-  const deps = pkgJson[`dependencies`];
-  const devDeps = pkgJson[`devDependencies`];
-
-  for (p in deps) {
-    pkgSet[p] = deps[p];
-  }
-  for (p in devDeps) {
-    devPkgSet[p] = devDeps[p];
-  }
-}
+directories.forEach((dir) => {
+  pkgJson = JSON.parse(fs.readFileSync(path.join(PROJECT_ROOT, dir, `package.json`)));
+  Object.assign(pkgSet, pkgJson.dependencies);
+})
```

Comment:
Missing semicolon. I don't think the lint script is seeing this file. We'll need to update the lint script to look at `scripts/*`